package.json

```diff
@@ -55,7 +55,7 @@
     "depcheck": "^1.4.3",
     "eslint": "^8.44.0",
     "eslint-config-prettier": "^8.8.0",
-    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-import": "~2.26.0",
```

Comment:
Can we go ahead and bump to `^2.29.0`? It looks like there is a fix for something in 2.28.1, but I'm not sure if it's what you're referring to: https://github.com/import-js/eslint-plugin-import/blob/main/CHANGELOG.md#fixed

Reply:
I checked `2.29.0` but it does not fix the issue. There are several open issues about it:

- import-js/eslint-plugin-import#2909
- import-js/eslint-plugin-import#2683
- import-js/eslint-plugin-import#2682
- And more...

It looks like this PR will fix it: import-js/eslint-plugin-import#2721

Reply:
I see. Thanks for following up!